extension/src/experiments/processExecution/index.ts

```diff
@@ -0,0 +1,23 @@
+import { join } from 'path'
+import { EXP_RWLOCK_FILE } from '../../cli/dvc/constants'
+import { getPidFromFile } from '../../fileSystem'
+import { Toast } from '../../vscode/toast'
+import { processExists, stopProcesses } from '../../processExecution'
+
+export const stopWorkspaceExperiment = async (dvcRoot: string) => {
+  const pid = await getPidFromFile(join(dvcRoot, EXP_RWLOCK_FILE))
+  if (!pid) {
+    return
+  }
+  if (!(await processExists(pid))) {
+    return
+  }
+
+  void Toast.showOutput(
+    stopProcesses([pid]).then(stopped =>
```

Comment:
Spotted something weird while testing though it's probably just a local issue (wouldn't be the first time 😅). I was on the `main` branch but when I stopped a running experiment and went back to my terminal, my terminal was checked out to a different position. 

https://user-images.githubusercontent.com/43496356/217856044-4aa3ec67-7d8c-4a54-a96b-e546ff032a22.mov

Reply:
This is what happens when you stop a checkpoint experiment. HEAD is being moved during the experiment.